felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -2150,6 +2150,7 @@ func mergeAttachPoints(ap4, ap6 attachPoint) attachPoint {
 				aptcV4.HostTunnelIPv6 = aptcV6.HostTunnelIPv6
 				aptcV4.HookLayoutV6 = aptcV6.HookLayoutV6
 				aptcV4.PolicyIdxV6 = aptcV6.PolicyIdxV6
+				aptcV4.IPv6Enabled = true
```

Comment:
Some leftover? Should we remove the `IPv6Enabled` from the tc endpoint completely? I think this was introduced years ago when we thought that we would do things differently. And I think we should also remove the flag from globals.